srtcore/queue.cpp

```diff
@@ -282,17 +283,17 @@ void CSndUList::update(const CUDT *u, EReschedule reschedule)
 
         if (n->m_iHeapLoc == 0)
         {
-            n->m_llTimeStamp_tk = 1;
+            n->m_tsTimeStamp = steady_clock::now();
```

Comment:
Still wrong.

Reply:
More hints, please?

Reply:
This `1` here was a value that was intended to satisfy the following conditions:
* It's not `0` (so it will fail the comparison test against `steady_clock()`)
* It's older than any imaginable time taken from `now()` or calculated therefrom by any reasonably big duration

Please analyze the code around this and you'll know why this is important.

Reply:
That is correct. The `1` was used to increase the priority of the socket in the heap. The same will do `now()`, except for the priority will not be increased over a socket, that had to be updated before the current time. Which makes more sense.

Reply:
I'm simply not sure if this change of the current behavior is correct, even if it seems to make more sense.